src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -107,30 +106,8 @@ protected override ILspServices ConstructLspServices()
 
         services.AddSingleton<IClientConnection>(_clientConnection);
 
-        if (_logger is LoggerAdapter adapter)
-        {
-            services.AddSingleton<LoggerAdapter>(adapter);
-        }
-        else
-        {
-            services.AddSingleton<LoggerAdapter>(static (provider) =>
-            {
-                var loggers = provider.GetServices<ILogger>();
-                if (!loggers.Any())
-                {
-                    throw new InvalidOperationException("No loggers were registered");
-                }
-
-                var telemetryReporter = provider.GetRequiredService<ITelemetryReporter>();
-                return new LoggerAdapter(loggers, telemetryReporter);
-            });
-        }
-
+        // Add the logger as a service in case anything in CLaSP pulls it out to do logging
```

Comment:
where did the logic get moved (replaced) now that `services.AddSingleton` etc is removed here?

Reply:
Nowhere, its just deleted. Nobody pulled the LoggerAdapter out of DI, so no point making it available